cesium_app/handlers/prediction.py

```diff
@@ -114,14 +124,18 @@ def post(self):
 
     def get(self, prediction_id=None, action=None):
         if action == 'download':
-            prediction = cesium.featureset.from_netcdf(self._get_prediction(prediction_id).file.uri)
-            with tempfile.NamedTemporaryFile() as tf:
-                util.prediction_to_csv(prediction, tf.name)
-                with open(tf.name) as f:
-                    self.set_header("Content-Type", 'text/csv; charset="utf-8"')
-                    self.set_header("Content-Disposition",
-                                    "attachment; filename=cesium_prediction_results.csv")
-                    self.write(f.read())
+            pred_path = self._get_prediction(prediction_id).file.uri
+            fset, data = featurize.load_featureset(pred_path)
+            result = pd.DataFrame({'ts_name': fset.index,
+                                   'label': data['labels'],
+                                   'prediction': data['preds']},
+                                  columns=['ts_name', 'label', 'prediction'])
+            if data.get('pred_probs'):
+                result['probability'] = np.max(data['pred_probs'], axis=1)
+            self.set_header("Content-Type", 'text/csv; charset="utf-8"')
```

Comment:
Probably! But I'd rather keep that separate since this is really only touching the `cesium`-related functionality.